homework/index.js

```diff
@@ -30,18 +32,129 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
-      }
+  function createContainer(selectedRepository) {
+    const container = document.getElementById('container');
+    const leftDiv = createAndAppend('div', container, {
+      id: 'leftDiv',
+      class: 'left-div white_Frame',
+    });
+    const table = createAndAppend('table', leftDiv);
+    const tbody = createAndAppend('tbody', table, { id: 'tbody' });
+
+    if (selectedRepository.name !== null) {
+      const tr1 = createAndAppend('tr', tbody);
+      createAndAppend('td', tr1, { class: 'label', text: 'Repository :' });
+      const td2 = createAndAppend('td', tr1);
+      createAndAppend('a', td2, {
+        href: selectedRepository.html_url,
+        target: '_blank',
+        text: selectedRepository.name,
+      });
+    }
+    if (selectedRepository.description !== null) {
+      const tr2 = createAndAppend('tr', tbody);
+      createAndAppend('td', tr2, { class: 'label', text: 'Description :' });
+      const repositoryDescription = createAndAppend('td', tr2, { text: '' });
+      repositoryDescription.textContent = selectedRepository.description;
+    }
+    if (selectedRepository.forks !== null) {
+      const tr3 = createAndAppend('tr', tbody);
+      createAndAppend('td', tr3, { class: 'label', text: 'Forks :' });
+      const fork = createAndAppend('td', tr3, { text: '' });
+      fork.textContent = selectedRepository.forks;
+    }
+    if (selectedRepository.updated_at !== null) {
+      const tr4 = createAndAppend('tr', tbody);
+      createAndAppend('td', tr4, { class: 'label', text: 'Updated :' });
+      const updated = createAndAppend('td', tr4, { text: '' });
+      updated.textContent = new Date(selectedRepository.updated_at).toLocaleString();
```

Comment:
Why not just put the `new Date(...)` statement directly in the `options` object you pass to `createAndAppend`?